test/Tests.hs

```diff
@@ -104,10 +104,12 @@ cabalGoldenTest name outRef act = goldenTest name readGolden (transform <$> act)
 
     upd (packData -> contents) = BS.writeFile outRef contents
 
-    cmp x y | x == y = return Nothing
-    cmp out1 out2 = return . Just . mconcat $
-        [ diff out1 out2
-        ]
+    cmp :: [String] -> [String] -> IO (Maybe String)
```

Comment:
This change is unnecessary.

Reply:
Indeed, but so is a redex `concat [x]` instead of simply `x`.  I improved code comprehension here.

Reply:
Do you want your redex back?

Reply:
The redex is an artifact of recent change: https://github.com/haskell-CI/haskell-ci/commit/6725ab5c

Cleaning it up is welcome, but I would prefer it done separately, it's not related to `--jobs-selections` change.

Reply:
I see.